keel-igor/src/main/kotlin/com/netflix/spinnaker/keel/services/ArtifactMetadataService.kt

```diff
@@ -95,18 +95,23 @@ class ArtifactMetadataService(
     val retry = Retry.of(
       "get artifact metadata",
       RetryConfig.custom<List<Build>?>()
-        .maxAttempts(10)
-        .waitDuration(Duration.ofMillis(1000))
-        .retryOnResult{
-          result -> result.isNullOrEmpty()
+        // retry 20 times over a total of 90 seconds
```

Comment:
@emjburns Does a minute and a half worth of additional waiting sound acceptable to you (worst case scenario)?